server/src/clj_kondo/lsp_server/impl/server.clj

```diff
@@ -114,14 +115,20 @@
                 (.getParentFile))]
     (config-dir dir)))
 
+(defn config-options
+  [args path]
+  (str/join "/" (concat [path] [(first args)])))
```

Comment:
- [ ] Do we want to support plain EDN config as a string in the VSCode input field as well? This function doesn't account for that.
- [ ] What if the path is absolute?
- [ ] Don't use `str/join` with `"/"` for creating paths, but rather `(io/file ... ...)`, else it won't work on Windows

Reply:
Will update with io/file and good point I think we want to support absolute or relative. For supporting plain EDN, should I just check to see if the string can be read in by EDN?